src/network/network.cpp

```diff
@@ -64,9 +64,10 @@ void send_and_receive_commands(sys::state& state) {
 		while(c) {
 			if(command::is_console_command(c->type)) {
 				// propagate to all the worker's queues
-				for(uint32_t i = 0; i < max_clients; i++)
-					if(state.network_state.clients[i].active)
-						state.network_state.clients[i].worker_commands.push(*c);
+				for(auto& client : state.network_state.clients)
+					if(client.active) {
+						// TODO: send commands to client
+					}
 				state.incoming_commands.push(*c); // and our local command queue
```

Comment:
you need to execute the command here, not store it. A command may affect the validity of the next command